NI.package.cmake

```diff
@@ -0,0 +1,195 @@
+if (TARGET ni-media)
```

Comment:
You may want to add a disclaimer as [here in ni-midi2](https://github.com/midi2-dev/ni-midi2/blob/main/NI.package.cmake#L1)

Reply:
Done.